apps/api/src/app/notification-template/usecases/create-notification-template/create-notification-template.usecase.ts

```diff
@@ -7,12 +7,16 @@ import { CreateNotificationTemplateCommand } from './create-notification-templat
 import { ContentService } from '../../../shared/helpers/content.service';
 import { CreateMessageTemplate } from '../../../message-template/usecases/create-message-template/create-message-template.usecase';
 import { CreateMessageTemplateCommand } from '../../../message-template/usecases/create-message-template/create-message-template.command';
+import { CreateChangeCommand } from '../../../change/usecases/create-change.command';
+import { CreateChange } from '../../../change/usecases/create-change.usecase';
+import { ChangeEntityTypeEnum } from '../../../../../../../libs/dal/src/repositories/change/change.entity';
```

Comment:
```suggestion
import { ChangeEntityTypeEnum } from '@novu/dal';
```
We should only import from the root libraries and not from the src areas. If this enum will be used in the client aswell we can definitely have it in @novu/shared instead. Wdyt?